app/javascript/vue/repository_item_sidebar/RepositoryItemSidebar.vue

```diff
@@ -273,14 +276,13 @@ export default {
       });
     },
     generateBarCode(text) {
-      if (!text) return
-      // generate the QR code
+      if(!text) return;
       const barCodeCanvas = bwipjs.toCanvas('bar-code-canvas', {
         bcid: 'qrcode',
         text,
         scale: 3
       });
-      this.barCodeSrc = barCodeCanvas.toDataURL('image/png')
+      $('#repository-item-sidebar #bar-code-image').attr('src', barCodeCanvas.toDataURL('image/png'));
```

Comment:
We shouldn't use jQuery if possible.
In this case, the previous implementation (`<img :src="barCodeSrc" />`) was better. Was there a specific reason for the switch to setting the attribute via jQuery?

Reply:
@artoscinote 
My thought here was to avoid a rerender of the view component and use jQuery instead!
And yea, the previous implementation is better if we should avoid the use of jQuery